### PR TITLE
Bump : slf4j-ext, gson, netty

### PR DESCRIPTION
**Pull request by NomadXD**
### Purpose
Bump : slf4j-ext, gson, netty

### Issues

Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -476,6 +476,11 @@
                 <version>${junit.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```
Comment (slahirucd7) [inline pom.xml:479]: If possible, shall we add a comment similar to this ?
https://github.com/wso2/product-microgateway/blob/main/pom.xml#L552

Comment (NomadXD) [reply pom.xml]: Comment added

**Review by VajiraPrabuddhaka — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE2 cluster : dev-deployment-v2 : 20220829.2

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20220829.2

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : dev-deployment-v2 : 20220829.2

**Comment by choreo-cicd** [comment]
[succeeded]  : dev-deployment-v2 : 20220829.3

**Comment by choreo-cicd** [comment]
[failed] DataplaneE1 cluster : stage-deployment-v2 : 20220905.1

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : stage-deployment-v2 : 20220905.2

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20220905.2

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : prod-deployment-v2 : 20220907.1

**Pull request merged**